packages/web-components/fast-foundation/src/menu/menu.ts

```diff
@@ -222,21 +222,22 @@ export class Menu extends FoundationElement {
         }
 
         function elementIndent(el: HTMLElement): MenuItemColumnCount {
-            if (!(el instanceof MenuItem)) {
-                return 1;
+            const role = el.getAttribute("role");
+            if (!role) {
```

Comment:
I'm not sure we want to be supporting things which do not have a role. If the role is set to menuitem, it should behave and slot as such.

Reply:
I guess we support divider here and that goes full width. I think the guiding principle I'd want to see here is that things are aligned *as menu items* only so long as they have the role of `menuitem`. Hopefully that clears my comment up enough :)

Reply:
I removed the check since all the elements processed here are guaranteed to have a role of 'menuitem', 'menuitemcheckbox', or 'menuitemradio'.